### PR TITLE
Remove JSHint completely.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

**Comment by XhmikosR** [comment]
@feimosi: can we get this merged?

File: gulpfile.js [hunk 1]
```diff
@@ -50,12 +50,12 @@ gulp.task('build.demo-js', function () {
...
 gulp.task('build.dist-css', function() {
     return gulp.src(src.css)
-        .pipe(plugins.if(/.scss/, plugins.sass({ style: 'compressed', noCache: true }))) // jshint ignore:line
+        .pipe(plugins.if(/.scss/, plugins.sass({ style: 'compressed', noCache: true })))
         .pipe(plugins.autoprefixer(autoprefixerBrowsers))
         .pipe(plugins.concat('baguetteBox.css'))
         .pipe(gulp.dest(dist.css))
         .pipe(plugins.concat('baguetteBox.min.css'))
-        .pipe(plugins.cssmin())
+        .pipe(plugins.cssmin({ compatibility: 'ie8' }))
```
Comment (XhmikosR) [inline gulpfile.js]: Should this be `ie7`?

**Comment by XhmikosR** [comment]
@feimosi: I went ahead and updated all devDependencies and made any needed changes. Please, don't leave this hanging for a long time.

**Comment by feimosi** [comment]
@XhmikosR Sorry, I've got some severe health issues. I'll get back to these slowly as soon as I can.

**Comment by XhmikosR** [comment]
@feimosi: sorry to hear that :( Take care and get well soon! All this mean nothing without our health.

**Comment by feimosi** [comment]
Thanks for a good word! I really hope I'll get better soon.

File: gulpfile.js (hunk 1 above)
Comment (feimosi) [reply gulpfile.js]: It's fine, we're officially supporting IE8+

**Comment by feimosi** [comment]
Sorry it took so long!

**Pull request merged**